src/item.ts

```diff
@@ -52,6 +53,7 @@ interface Fields extends Format {
 	internalShortId: Option<string>;
 	commentCount: Option<number>;
 	relatedContent: Option<ResizedRelatedContent>;
+	footballContent: Option<FootballContent>;
```

Comment:
One thing we _could_ do is:

```ts
interface MatchReport extends Fields {
  design: Design.MatchReport;
  body: Body;
  football: Option<FootballContent>;
}
```

since the football data is only relevant to match reports.